src/System.Collections.Immutable/tests/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard;
+      netstandard1.3;
```

Comment:
If you do what I suggested, including the code (defining the constant) when TargetGroup != netstandard1.0 then you don't need to do this for the tests.